etc/ci/performance/README.md

```diff
@@ -64,9 +64,10 @@ If you want to test the data submission code in `submit_to_perfherder.py` withou
 ## For Gecko
 
 * Install Firefox Nightly in your PATH
-* Install [jpm](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm#Installation)
-* Run `jpm xpi` in the `firefox/addon` folder
-* Install the generated `xpi` file to your Firefox Nightly
+* Download [geckodrive](https://github.com/mozilla/geckodriver/releases) and add it to the `PATH`
+* `pip install selenium`
+* Run `python gecko_driver.py` to test
```

Comment:
I'm curious. Do we really need geckodriver for selenium to work with firefox? I'm sure we need chromedriver for chrome, but I thought selenium supported firefox without the driver.